README.md

```diff
@@ -17,8 +17,7 @@ const gen = immutagen(function*() {
   yield 1
   yield 2
   return 3
-})
-gen.next()                // { value: 1, next: [function] }
+})()                      // { value: 1, next: [function] }
 gen.next()                // { value: 1, next: [function] }
```

Comment:
You're right also about the comment, this line's comment should be `// { value: 2, next: [function] }`

Reply:
Fixed it. Also added newlines to highlight different "steps" in the explanation below.